pkg/network/cache/types.go

```diff
@@ -9,10 +9,19 @@ import (
 	v1 "kubevirt.io/client-go/api/v1"
 )
 
+type PodIfaceState int
```

Comment:
Are you sure this is the correct commit message?
I remember writing it on another PR/commit:)

Reply:
As asked by @maiqueb the PR commits have being squashed.